lib/vsc/utils/script_tools.py

```diff
@@ -42,7 +42,7 @@
 from vsc.utils.generaloption import SimpleOption
 from vsc.utils.lock import lock_or_bork, release_or_bork, LOCKFILE_DIR, LOCKFILE_FILENAME_TEMPLATE
 from vsc.utils.nagios import (
-    SimpleNagios, NAGIOS_CACHE_DIR, NAGIOS_CACHE_FILENAME_TEMPLATE, exit_from_errorcode,
+    SimpleZabbix, SimpleNagios, NAGIOS_CACHE_DIR, NAGIOS_CACHE_FILENAME_TEMPLATE, exit_from_errorcode,
```

Comment:
why do you import this?

Reply:
indeed not needed, fixed in https://github.com/hpcugent/vsc-utils/pull/81/commits/3f0e291587fb40b17edb9a946df04053a418d697